### ISSUE TITLE
WoW Classic Era V1.15.1 latest release no longer working (03/16/2024)

### ISSUE THREAD

**Issue by maximus210793** [DeadlyBossMods/DBM-BCVanilla#108]
Hi,
I am playing WoW Classic Era V1.15.1.
Since the latest release DBM is no longer opening ingame (Release: (03/16/2024))
Thanks for fixing it.

**Comment by MysticalOS** [comment]
it's possible you have an alpha version of raid and dungeon modules but don't have latest core yet due to fact it's not packaging correctly yet. the last release version of everything should be stable but yeah i suspect you aren't on that if it's not working.

the alphas SHOULD be stable again soonish as well.

**Comment by MysticalOS** [comment]
Since this is now live. Posting further clarification below:

Completion of mod unification:

With latest updates, DBM has completed long process of making every module a hybrid mod that auto loads to conditions and needs of current client running from. As such, no custom modules exist for specific game versions any more. Every DBM mod is designed to "just work" regardless of what addons folder it's thrown into.

You can basically copy any DBM addon from classic to retail, or other way around, any flavor and it auto loads/changes to code base needed for that game version.

This also reduces issues with mod managers going forward when multiple versions/copies of same module exist in diff packages, such as wrath classic package also containing same modules as wrath retail package. Distributing same files in two diff packages can cause problems. This also lead to user confusion more often than not. A user would search for DBM wrath and find the retail modules for it and not the classic ones.

Going forward, the retail ones are also the classic ones. The only caveat is that if you download just deadly boss mods core, you don't actually have raid modules for cataclysm, wrath, tbc, or vanilla until you also download the packages for those modules. I've decided this is still better with all the perks and confusion that it eliminates after a brief confusion for users who one minute had DBM Core+Custom wrath modules in a single package to DBM Core+Unified wrath modules as two separate downloads.

TL/DR. 
The modules that work on retail, also work on cata, wrath, tbc, or vanilla classic.
For retail Download DBM Core and DBM Dungeons and you're good to go. For cataclysm Download DBM Core + DBM Cataclysm + DBM Dungeons (optionally can also download wrath, tbc and vanilla modules for cataclysm too that's up to you). Wrath and vanilla is same thing. Core + Game version package + Dungeons (if you want em).
Easier more targeted updating.

**Comment by maximus210793** [comment]
Thanks mate for the feedback I additionally installed:
DBM - TBC & Vanilla (Requires Deadly Boss Mods)
now together with:
Deadly Boss Mods (DBM)
DBM - Dungeons & Events (Requires Deadly Boss Mods)
DBM - PvP Mods (Requires Deadly Boss Mods)
It works fine for WoW Classic Era V 1.15.1

**Comment by MysticalOS** [comment]
yep, sorry for confusion, i checked and htere were bugs that prevented the in game messages from showing, especially for SoD gnomer so users weren't seeing message about the mod restructure change

**Comment by maximus210793** [comment]
are there any other useful extensions for WoW Classic Era (not SOD) I should install ?

**Comment by seunggi5580** [comment]
> yep, sorry for confusion, i checked and htere were bugs that prevented the in game messages from showing, especially for SoD gnomer so users weren't seeing message about the mod restructure change

Suddenly, classic hardcore servers don't work
There is no notification in the Blackwing Lair
Which version should I roll back?

**Comment by MysticalOS** [comment]
Please read like 2 comments up, https://github.com/DeadlyBossMods/DBM-BCVanilla/issues/108#issuecomment-2009060426
